packages/stable/src/__tests__/api/datamodels.int.spec.ts

```diff
@@ -48,8 +48,17 @@ describe('Data models integration test', () => {
 
   beforeAll(async () => {
     jest.setTimeout(30 * 1000);
-
     client = setupLoggedInClient();
+
+    const spaces = await client.spaces
+      .list({ limit: 1000 })
+      .autoPagingToArray();
+    const oldSpaces = spaces.filter(
+      // Filter spaces last updated more than 24 hours ago
+      (space) => space.lastUpdatedTime < Date.now() - 24 * 60 * 60 * 1000
+    );
+    await client.spaces.delete(oldSpaces.map((space) => space.space));
```

Comment:
Move to a function to avoid duplicate code?

Reply:
Good suggestion 
will do that